<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>365</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Log X</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TextLableLayout">
     <item>
      <layout class="QVBoxLayout" name="LabelLayout">
       <item>
        <widget class="QLabel" name="RFIDLabel">
         <property name="text">
          <string>RFID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="PeriodLabel">
         <property name="text">
          <string>Period</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="PaymentLabel">
         <property name="text">
          <string>Payment</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="TextLayout">
       <item>
        <widget class="QLineEdit" name="RFIDText">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NameText">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PeriodText">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PaymentText">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>188</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="QuitLayout">
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>155</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="SerialButton">
       <property name="text">
        <string>Serial port setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UserSettingButtton">
       <property name="text">
        <string>User setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DatabaseButton">
       <property name="text">
        <string>Database setting</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>69</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuestionButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/logo/logo/qestion.png</normaloff>:/logo/logo/qestion.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string> software</string>
    </property>
   </widget>
   <widget class="QLabel" name="PictureLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>71</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
